tests/js/jest.config.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
     '<rootDir>/tests/js/setup-globals',
     '<rootDir>/tests/js/setup-mocks',
     'jest-canvas-mock',
+    'core-js',
```

Comment:
Curious: why is that needed? Because of the datepicker component?

Reply:
Yep, `flat` fails otherwise somewhere, for me at least.